SAPex/Controllers/SandboxesController.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using BusinessLogicLayer.DtoModels;
+using BusinessLogicLayer.Interfaces;
+using BusinessLogicLayer.Services;
+using DataAccessLayer.Service;
+using DbMigrations.Data;
+using FluentValidation.Results;
+using Microsoft.AspNetCore.Mvc;
+using PagedList;
+using SAPex.Mappers;
+using SAPex.Models;
+using SAPex.Models.Validators;
+
+namespace SAPex.Controllers
+{
+    [Route("api/sandbox")]
```

Comment:
The name "Sandboxes" of the controller is fine. But in the route, we have "sandbox" still. Please update the route name.

Reply:
done.